### ISSUE TITLE
field doesn't appear

### ISSUE THREAD

**Issue by huvber** [aldeed/meteor-cfs-autoform#30]
I'm trying to use it with autoform but the fields doesn't appear. Just the label. 
```javascript
CTIcons = new FS.Collection('ctypeicons',{
  stores: [ new FS.Store.GridFS('ctiStore')]
});
CTypes = new Meteor.Collection('ctypes');
CTypes.attachSchema(new SimpleSchema({
  ...
  icon: {
    type: String,
    label:'Icon',
    autoform: {
      afFieldInput: {
        type: "cfs-file",
        collection: "ctypeicons"
      }
    }
  }
}));

...
AutoForm.setDefaultTemplateForType('quickForm', 'plain');

```
in the template I have

```html
<template name="cTypeEdit">
    ...
    {{> quickForm collection="CTypes" doc=this }}
    ...
  </div>
</template>
```

The result is this:
![schermata da 2015-03-06 11 53 39](https://cloud.githubusercontent.com/assets/5032333/6524257/edbfd9f4-c3f7-11e4-977e-f125b156d9fd.png)

Am I missing something?

**Comment by aldeed** [comment]
It may be because of setting the QF template to "plain". There is only a bootstrap template for the input right now.

**Comment by huvber** [comment]
I tried to put the template field on the schema but doesn't change anything. Unfortunately I'm using foundation so I can't use the bootstrap theme. Could you please tell me where you generate the html code for the field so I can add the part for the plain template

**Comment by huvber** [comment]
I also tried removing 
```javascript
AutoForm.setDefaultTemplateForType('quickForm', 'plain');
```
but it doesn't change anything

**Comment by ntoka** [comment]
I'm having the same problem. Adding bootstrap3 as template does not help. I'm using autoforms with meteoric:autoforms-ionic

**Comment by huvber** [comment]
I tried with the new version and I still have this issue. I want also to report that if I using 
```handlebars
{{> afQuickField name="pkg" type="cfs-file" collection="cbins" template="bootstrap3" }}
```
I can see the input tag but it will show this error
```
Exception from Tracker afterFlush function: undefined is not a function
TypeError: undefined is not a function
    at Template.cfsFileField_bootstrap3.rendered.Template.cfsFilesField_bootstrap3.rendered (http://localhost:3000/packages/cfs_autoform.js?92ec069087406a8d536c1ee5147784343469bf55:390:22)
    at http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3061:22
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3382:12)
    at fireCallbacks (http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3057:12)
    at null.<anonymous> (http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:3150:5)
    at http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1734:14
    at Object.Blaze._withCurrentView (http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:2105:12)
    at http://localhost:3000/packages/blaze.js?4e49999979a58da0e2265f7bd3f5910f9901b07b:1733:15
    at Tracker.flush (http://localhost:3000/packages/tracker.js?21f0f4306879f57e10ad3a97efe9ea521c5b5775:460:11)
```

**Comment by aldeed** [comment]
Created #33 to track the enhancement request. The error should be fixed in 2.1.2 release.
